config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   #root "users#index"
```

Comment:
I highly recommend fixing the routes since when I open the app I should see signup form so the user can start entering his information but I should add it in the URL users/sign_up so please add them to your routes so when the user run the app he can see the signup form immediately ♻🔥🙏🏻


![image](https://github.com/AlaaAlsalem/blogaapp/assets/58553711/103b2152-bfff-4850-9955-dc577e60394a)